my.css

```diff
@@ -352,3 +352,157 @@ body {
     justify-content: center;
   }
 }
+
+/* Вторая секция */
+.bagscluth__content {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+.bagscluth {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  max-height: 900px;
+  display: flex;
+  background-color: #fae8e4;
+}
+.bagscluth__img {
+  width: 60%;
+  height: 844px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+.bags-img {
+  position: absolute;
+  display: block;
+  width: 350px;
+  height: 380px;
+  padding-left: 100px;
+
+}
+.bags-img1 {
+  position: absolute;
+  width: 550px;
+  height: 750px;
+  left: 190px;
+}
+
+.inner {
+  width: 400px;
+  position: relative;
+  padding: 30px;
+}
+.card {
+  position: absolute;
+  width: 400px;
+  height: 456px;
+  padding: 40px 55px 45px 65px; 
+  margin: 0 auto;
+  top: 200px;
+  display: flex;
+  flex-flow: column;
+  background-color: #fff;
+  border: 1px solid #fff;
+  box-shadow: 29px 30px 22px 0px rgba(0,0,0,0.07);
+}
+
+
+.card__text {
+ padding-top: 15px;
+ padding-bottom: 25px;
+ margin: 0 auto;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)